test/api

```diff
@@ -211,7 +211,7 @@ step_7_test_meta()
 
     shakedown GET "$flipt_host/meta/config"
         status 200
-        contains "\"logLevel\""
```

Comment:
It looks like this file could be removed since it has no references to it. Is it okay to remove this, or is there a reason behind it being here @markphelps ?

Reply:
good call! I thought I removed it. I think you can go ahead and remove the entire `test` directory